prisma/seed.ts

```diff
@@ -52,6 +100,37 @@ async function main() {
             },
             update: {},
         }),
+        prisma.authUser.upsert({
+            where: {
+                id: 'test_f5ec79b4-8c49-4015-bc37-f2758923ef38',
+            },
+            create: {
+                id: 'test_f5ec79b4-8c49-4015-bc37-f2758923ef38',
+                firstName: 'Karan',
+                lastName: 'Handa',
+                email: 'test@maybe.com',
+                password: hashedPassword,
+            },
+            update: {},
+        }),
+        prisma.user.upsert({
+            where: {
+                authId: 'test_f5ec79b4-8c49-4015-bc37-f2758923ef38',
+            },
+            create: {
+                email: 'test@maybe.com',
+                firstName: 'Karan',
+                lastName: 'Handa',
+                authId: 'test_f5ec79b4-8c49-4015-bc37-f2758923ef38',
+                onboarding: onboarding,
+                household: 'single',
+                country: 'US',
+                maybe: 'Test',
+                createdAt: new Date('2024-01-20T15:21:17.889Z'),
```

Comment:
Does this need to be set? I think the db will handle this.

Reply:
Done.